### ISSUE TITLE
Error in Model and Problem with Embedded Writing - FlashPrint

### ISSUE THREAD

**Issue by BlackMeister** [paulhayes/web-3d-model-customiser#39]
Hi, thanks for doing this.  Used a few days ago and all worked well.  Over the last couple of days I have noticed issues when I load the model into FlashPrint (for FlashForge) with the indent writing that was fine before, but now loads strangely and mainly fills in and has strange line artefacts over the wording when FlashPrint fixes the model. This happens after I get an "Error has occurred to the model", if I don't fix when I go to print you can see missing layers within the print. When I do fix it, it sorts the missing layers but messes up the writing.  Just loaded old models (which can't use as they have old dates embedded) and while it still asks to repair the model, the embedded writing is fine and the model prints fine.

**Comment by sebleedelisle** [comment]
Can you change your layer height to 0.25mm and see if it still happens?

**Comment by sebleedelisle** [comment]
Similar to #28

**Comment by BlackMeister** [comment]
So it's strange.  Already set to 0.25.  However, after I select print to save the file, then I look at the pre print model, with 'PETG' on the right and say my name on the left.  On the left the name is still messed up / filled in.  On the right the bottom layer only shows the letter 'P' and the remain layers (selected 6) have 'ETG' with 'P' missing.  If I don't repair the model it's the same, lots of errors of missing layers.

**Comment by BlackMeister** [comment]
It's your discussion forum, but not sure my issue is the same as #28.

**Comment by sebleedelisle** [comment]
Sorry not sure what happened there.

**Comment by BlackMeister** [comment]
The same problem of the error and the embedded writing occurs with 1 in the stack.

**Comment by sebleedelisle** [comment]
Unfortunately there's an issue with the underlying OpenJsCad lib that produces non-manifold models, and I don't think FlashPrint can fix them. You can try fixing the models in Netfabb maybe? Or else send me an STL and I'll make sure that there's nothing else going on.

**Comment by BlackMeister** [comment]
Thanks.  Tried playing around with PrusaSlicer, but still get some errors when back in FlashPrint and not sure FlashForge accepts G-Code.  If you could upload a reapired six stack with no date and no info that would be great.

**Comment by sebleedelisle** [comment]
You can export an STL from PrusaSlicer once it's fixed, which you can then
load into FlashPrint

**Comment by BlackMeister** [comment]
Did that, as an stl and seperatly an obj.  Nearly all repaired, but there was still a missing layer in the bottom mask.

**Comment by sebleedelisle** [comment]
Yeah I'm seeing that sometimes too, but Netfabb seems to be able to fix
this error (although Paul is trying to get to the bottom of this on the
OpenJsCad side as we speak, so fingers crossed!)

**Comment by BlackMeister** [comment]
Managed to get a 5 stack to work via Prusa, this is new to me so thanks for bringing that skill. I have a Mac so think Netfabb only MS?

**Comment by sebleedelisle** [comment]
Ah that's cool 😊 I never downloaded the Netfabb app but there's an online
version you can try.

**Comment by BlackMeister** [comment]
Just to let you know.  No matter what I do in repairing the file in Prusa, it all looks fine when I import it and is, except when I then slice/save the file in FlashPrint, it seems to add a raft even when a raft is not selected.
